ee/codegen/src/generators/workflow.ts

```diff
@@ -754,10 +754,16 @@ class WorkflowDisplayFile extends BasePersistedFile {
 
   public getModulePath(): string[] {
     if (this.workflowContext.parentNode) {
-      return [
-        ...this.workflowContext.parentNode.getNodeDisplayModulePath(),
-        GENERATED_WORKFLOW_MODULE_NAME,
-      ];
+      return this.workflowContext.nestedWorkflowModuleName
```

Comment:
[oos] I would expect a workflow context with a parent node would always have a nested workflow module name. not worth debugging now, but worth watching and seeing if we could cleanup in the future